examples/appliance_device_snmp_v3_users.py

```diff
@@ -46,7 +46,7 @@
 
 # Add appliance device SNMP v3 users
 snmp_v3_user = oneview_client.appliance_device_snmp_v3_users.create(options)
```

Comment:
create an object for snmp_v3_user class and use that object for all other methods. Refer to other resources for reference.
```
snmp_v3_users = oneview_client.appliance_device_snmp_v3_users
snmp_v3_user = snmp_v3_users.create(options)
```